share/completions/run0.fish

```diff
@@ -0,0 +1,21 @@
+#
+# Completion for run0
```

Comment:
I have no idea what is run0

I would expect this comment to provide a link to the repository and a short explanation about what run0 is about